src/Take.Blip.Builder/Models/Flow.cs

```diff
@@ -204,5 +204,18 @@ public static Flow ParseFromJson(string json)
         /// <param name="filePath">The path.</param>
         /// <returns></returns>
         public static Flow ParseFromJsonFile(string filePath) => ParseFromJson(File.ReadAllText(filePath));
+
+        /// <summary>
+        /// Fully checks if a given config header should be added or not
+        /// </summary>
+        /// <param name="header">Config header to check for</param>
+        /// <returns>Boolean indicating if the configuration is enabled</returns>
+        public bool IsConfigurationFlagEnabled(string header)
```

Comment:
It's getting good!
Rename this parameter (and XML doc) to something meaningfull to this method, like `configurationKey`.
The local parameters `identifierHeaderValue` and `sendBotIdentifier` will need better names likewise (suggestions: `configurationValue` and `configurationFlag`)

Reply:
Changed!